### ISSUE TITLE
How to export L2-normalization to onnx

### ISSUE THREAD

**Issue by stoneyang** [pytorch/pytorch#32041]
## 🚀 Feature
Support export for LpNormalization from PyTorch to ONNX, thus it could be used in TensorRT model.

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by egbertYeah** [comment]
> ## 🚀 Feature
> Support export for LpNormalization from PyTorch to ONNX, thus it could be used in TensorRT model.
> 
> cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

I found the L2 normalization in pytorch with torch.nn.functional.normalize, but when i convert this op to onnx file with the function "torch.onnx", the result of onnx is not equal to LpNormalization. how can i solve this problem?

**Comment by garymm** [comment]
@stoneyang / @egbertYeah please provide clear repro instructions including the model code, the call to `torch.onnx.export` and the expected vs actual behavior.
If possible please try the latest PyTorch nightly version.
